vyos/vyos-build
# T4849: add Ten64/NXP DPAA2 support patches

## Types of changes

 

## Related Task(s)
https://phabricator.vyos.net/T4849

## Component(s) name


## Proposed changes
Updates the arm64 kernel configuration to 5.15 (equivalent to the x86 version at the time of writing), as well as the image data for arm64 (correct GRUB + kernel package variants). 
Additionally, patches to fix SFP+ issues on the Traverse Technologies Ten64 platform are included
(Note to self: To avoid breaking x86 in the future it could be a good idea to only apply these patches when target arch = arm64. I'll create a new task/pull if I get around to that)

## How to test
This is part of the 'port' I have done over here: https://github.com/mcbridematt/vyos-arm64-builder . I am slowly iterating down my patchset (there are various other bits that still need to be addressed for a complete x86-equivalent build)
See here for more info: https://forum.traverse.com.au/t/vyos-build-my-repo/181/5 or in vyos-on-arm64 in the VyOS slack

## Checklist:
